src/newSeed/stage1.ts

```diff
@@ -6,17 +6,17 @@ export class Stage1 extends BaseStage {
     let message: string;
 
     if (!this.seedConfig?.general?.projectName) {
-      message = "Enter a value for Project Name";
+      message = "Please Enter a value for Project Name";
     }
     else if (!Utils.isValidUrl(this.seedConfig?.general?.projectWebsite, false)) {
-      message = "Please enter a valid website url";
+      message = "Please enter a valid url for project website";
```

Comment:
Can you please use the same first-caps as the label text, all of these messages?

Reply:
Okay